### PR TITLE
Made Themes page more elaborative

### PR DESCRIPTION
**Pull request by thattheredstoner**

### CODE REVIEW

**Comment by Mubeen142** [comment]
Hi @thattheredstoner 

Thank you for the pull request.

Could u revert the changes with the web server using php 8.2, that is no longer needed as in the new update wemx supports 8.1 just fine.

**Comment by thattheredstoner** [comment]
Done

File: developers/themes.md [hunk 1]
```diff
@@ -19,13 +19,18 @@ Themes allow you to dynamically change the appearance of your panel without affe
...
-7. Duplicate the default Locales module interface folder `Modules/Locales/Resources/views/client/tailwind` to `Modules/Locales/Resources/views/client/material`
-8. Activate the new theme in the Admin Area
-9. You may customize the theme to your preference
+2. Inside the root of the newly created theme folder you will find a file called `theme.php` here you can specify the new name. Make sure to change the name to Material
+3. Duplicate the all the default service and module folders
+```
+e.g. app/Services/Universal/Resources/views/client/tailwind -> app/Services/Universal/Resources/views/client/material
+or
+Modules/Affiliates/Resources/views/client/tailwind to Modules/Affiliates/Resources/views/client/material
+```
```
Comment (PadowYT2) [inline developers/themes.md]:
```suggestion
3. Duplicate all the default service and module folders, for example
- app/Services/Universal/Resources/views/client/tailwind `->` app/Services/Universal/Resources/views/client/material
- Modules/Affiliates/Resources/views/client/tailwind `->` Modules/Affiliates/Resources/views/client/material
```

**Review by PadowYT2 — CHANGES_REQUESTED** [review]

**Comment by thattheredstoner** [comment]
Tweaked theme.md to match requested change and added chmod files to installation.md

File: project/installation.md [hunk 2]
```diff
@@ -156,6 +156,11 @@ chown -R nginx:nginx /var/www/wemx/*
 chown -R apache:apache /var/www/wemx/*
 ```
 
+Remember to also chmod your storage.
+```php
+chmod -R /var/www/wemx/storage/* /var/www/wemx/bootstrap/cache/
+```
+
```
Comment (PadowYT2) [inline project/installation.md]: WemX installer already does chmod, also the command is invalid

**Review by PadowYT2 — CHANGES_REQUESTED** [review]

**Comment by thattheredstoner** [comment]
Reverted addition of that

**Pull request merged**